server/src/database/queries/search.ts

```
@@ -0,0 +1,1198 @@
+  private applyScoreFusion(
+    results: EnhancedSearchResult[],
+    method: 'linear' | 'rank' | 'weighted'
+  ): EnhancedSearchResult[] {
+    switch (method) {
+      case 'linear':
+        // Scores are already combined linearly
+        return results
+
+      case 'rank':
+        // Use rank-based fusion (RRF - Reciprocal Rank Fusion)
+        const ranked = results.sort((a, b) => b.relevanceScore - a.relevanceScore)
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)